### PR TITLE
Feat: Move global args to the plan args to increase configurability

### PR DESCRIPTION
**Pull request by laurentluce**
Move global args to the plan args to increase configurability and remove some dead code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,54 @@ Simply run
...
+
+<details>
+    <summary>Click to see configuration</summary>
+
+You can configure this package using a JSON structure as an argument to the `kurtosis run` function. The full structure that this package accepts is as follows, with default values shown (note that the `//` lines are not valid JSON and should be removed!):
+
+```javascript
+{
+    // The number of nodes
+    "rabbitmq_num_nodes": 3,
```
Comment (laurentluce) [inline README.md:27]: Prefix the arg names with the package name to avoid name clashing when the args dictionary is passed to other packages.

**Review by adschwartz — APPROVED** [review]

**Comment by adschwartz** [comment]
@laurentluce looks great! Thank you. I'll leave it to you to merge.

**Comment by laurentluce** [comment]
> @laurentluce looks great! Thank you. I'll leave it to you to merge.

@adschwartz I don't think I have write access to this repository.

**Comment by adschwartz** [comment]
@laurentluce You should have now. Can you try again?

**Pull request merged**